popUp.css

```diff
@@ -0,0 +1,194 @@
+#overlay {
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  background: #ebf0ee;
+  width: 100vw;
+  height: 108vh;
+  opacity: 0.9;
+  z-index: 100;
+  display: none;
+}
+
+.popUp {
+  width: 90vw;
+  margin-left: 20px;
+  padding: 8px 0 0 0;
+  font-family: "inter", sans-serif;
+  border: 1px solid #dfe1e6;
+  box-sizing: border-box;
+  border-radius: 16px;
+  position: fixed;
+  top: 5px;
+  background-color: white;
+  z-index: 200;
+}
+
+.popUp-close {
+  color: white;
+  font-size: 18px;
+}
+
+.popUp-title {
+  width: 259px;
+  font-size: 32px;
+  color: #172b4d;
+  font-weight: bold;
+  margin-top: 2px;
+}
+
+.pop-pDesk {
+  display: none;
+  font-size: 16px;
+}
+
+.popUp-list {
+  display: flex;
+  font-size: 16px;
+  margin-left: -40px;
+  margin-top: -16px;
+}
+
+.popUp-items {
+  color: #3a4a42;
+  font-size: 12px;
+  background-color: #ebf0ee;
+  border-radius: 4px;
+}
+
+.popUp-descr {
+  font-size: 16px;
+  margin: 0 16px 20px 0;
+  color: #344563;
+}
+
+.lateral {
+  padding: 0 0 0 10px;
+  font-family: "inter", sans-serif;
+}
+
+.popup-image {
+  margin: auto;
+  width: 275px;
+  height: 220px;
+  background-image: url(./img/Mobile-popup-image.png);
+  background-repeat: no-repeat;
+}
+
+.desk {
+  display: none;
+}
+
+.popUp-btn {
+  display: flex;
+  justify-content: space-between;
+  margin: -10px 16px 0 0;
+  padding-bottom: 15px;
+}
+
+.p-btn {
+  width: 126px;
+  height: 48px;
+  background: #36b37f;
+  border-radius: 4px;
+  color: white;
+  font-size: 15px;
+  font-family: "inter", sans-serif;
+  border: none;
+}
+
+.live {
+  width: 14px;
+  height: 14px;
+}
+
+@media only screen and (min-width: 768px) {
+  #overlay {
+    position: fixed;
+    top: 0;
+    bottom: 0;
+    background: #ebf0ee;
+    width: 100vw;
+    height: 108vh;
+    opacity: 0.99;
+    z-index: 100;
+    display: none;
+  }
+
+  .popUp {
+    width: 62vw;
+    z-index: 200;
+    position: fixed;
+    overflow-y: scroll;
+    height: 100vh;
+    margin-left: 20%;
+  }
+
+  .popUp-close {
+    color: #67798e;
+    margin-top: 4px;
+    background-color: #ebecf0;
+    width: 20px;
+    height: 20px;
+    align-items: center;
+    padding-left: 4px;
+  }
+
+  .popup-image {
+    background-image: url(./img/Desktop-popup-image.png);
+    background-repeat: no-repeat;
+    width: 800px;
+    height: 600px;
+    margin-top: 30px;
+  }
+
+  .popUp-title {
+    margin-top: 20px;
+  }
+
+  .desk {
+    display: block;
+  }
+
+  .popUp-descr {
+    display: none;
+  }
+
+  .lateral {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .popUp-btn {
+    margin-top: 24px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    margin-left: -30px;
+  }
+
+  .p-btn {
+    width: 110px;
+    font-size: 14px;
+  }
```

Comment:
- In the desktop version kindly make sure this section shown in the picture below looks like it shows in the Figma design. The buttons in the popup for see live and see source are in the right position it should be better to push them a bit to the right so that it will make your design much more to the Figma design as it was required in the project :grinning: 

## Your design 

![image](https://user-images.githubusercontent.com/95297251/204062915-97d422af-b088-4d65-bb75-d2af3e14972e.png)

## Figma design 

![image](https://user-images.githubusercontent.com/95297251/204062973-ce29ef44-0e83-49e0-99a8-f8092307565c.png)